docs/v0.9/en/user-guide/query-data/view.md

```diff
@@ -0,0 +1,119 @@
+# View
+
+In SQL, a view is a virtual table based on the result set of an SQL statement. It contains rows and columns just like a real table.
```

Comment:
Here we explained what a View is.
we need to further explain why users need a View:
- What problem does the View solve for users
- In what scenarios do users need to user View

Reply:
Done